src/clients/Weechat.tsx

```diff
@@ -57,7 +57,18 @@ const Weechat: TextClient = {
                 return <span>Weechat doesn't support this kind of link</span>;
         }
     },
-    description: 'Commandline Matrix interface using Weechat',
+    copyString: (link) => {
+        switch (link.kind) {
+            case LinkKind.Alias:
+            case LinkKind.RoomId:
+                return `/join ${link.identifier}`;
+            case LinkKind.UserId:
+                return `/invite ${link.identifier}`;
+            default:
+                return '';
+        }
+    },
+    description: 'Command-line Matrix interface using Weechat',
```

Comment:
https://matrix-to-static-jyhgh1p1y.vercel.app/#/#matrix:matrix.org seems to still show the previous description (without hyphen), is there another string to change...?

Reply:
No, that build was based on my local checkout. I hadn't pulled the git commit